README.md

```diff
@@ -104,11 +107,37 @@ Read, Write, and Admin for Development Information (branches, commits, and pull
 
 #### GitHub permission scopes
 
+##### Repository Permissions
+
+|**Permission scope**|**Why the app needs it**|
```

Comment:
I'm not sure if this section needs a table.  I feel like it makes it harder to read when all the content is list based anyways

Reply:
It's currently a table on main. I've just added extra content. 

Did you look at the file? I don't think it's confusing.

<img width="1078" alt="Screen Shot 2022-01-18 at 9 19 20 am" src="https://user-images.githubusercontent.com/37155488/149842885-3a8feccf-db38-4e95-a373-5fe132130b56.png">